src/models/Studio.ts

```diff
@@ -13,7 +13,15 @@ const messageQueue: string[] = [];
 
 export const DEFAULT_PORT = 3210;
 
+function isValidFilePath(filePath: string): boolean {
+  return existsSync(filePath);
+}
+
 export function start(filePath: string, port: number = DEFAULT_PORT): void {
+  if (!isValidFilePath(filePath)) {
+    console.log('Invalid file path: ', filePath);
+    return;
```

Comment:
```suggestion
    throw new SpecificationFileNotFound(filePath);
```
I think it would be better to throw an error. Since we already have a error for this scenario we should reuse it. 
You can look for it here https://github.com/asyncapi/cli/blob/283de0f36c8bf934b9fd3337bf52ce44932a5e0d/src/errors/specification-file.ts#L9

I think it would be better to throw an error instead